packages/components/src/components/textarea/textarea.stories.ts

```diff
@@ -1,16 +1,18 @@
 import '../../solid-components';
 import { html } from 'lit-html';
 import { storybookDefaults, storybookHelpers, storybookTemplate } from '../../../scripts/storybook/helper';
-import { userEvent } from '@storybook/test';
-import { waitUntil } from '@open-wc/testing-helpers';
 import { withActions } from '@storybook/addon-actions/decorator';
 
 const { argTypes, args, parameters } = storybookDefaults('sd-textarea');
 const { generateTemplate } = storybookTemplate('sd-textarea');
 const { overrideArgs } = storybookHelpers('sd-textarea');
 
+/**
+ * Collect data from the user and allow multiple lines of text.
+ */
 export default {
   title: 'Components/sd-textarea',
+  tags: ['!dev'],
```

Comment:
Please remove help-text and show default version with no text inside

Reply:
@MartaPintoTeixeira Do you mean the tags? I am not sure what to check here :)

Reply:
@Vahid1919 pls mirror [figma](https://www.figma.com/design/YDktJcseQIIQbsuCpoKS4V/Component-Docs?node-id=2619-19572&t=xttBWX3tCRS19PNs-11) docs in code. THX

Reply:
Answered via call